hermeto/core/checksum.py

```diff
@@ -37,9 +38,37 @@ def __str__(self) -> str:
 
     @classmethod
     def from_sri(cls, sri: str) -> "ChecksumInfo":
-        """Convert the input Subresource Integrity value to ChecksumInfo."""
-        algorithm, checksum = sri.split("-", 1)
-        return ChecksumInfo(algorithm, base64.b64decode(checksum).hex())
+        """Convert the input Subresource Integrity value to ChecksumInfo.
+
+        The integrity string may contain multiple hashes separated by whitespace.
```

Comment:
I think it would be worth mentioning the SRI docs (maybe even the schema) to back this docstring.

Reply:
Might be also worth mentioning here what others could be because the SRI docs doesn't mention anything past SHA-2, but in fact it could be SHA-1 if git commits are referenced in that field.